quinn-proto/src/connection/stats.rs

```diff
@@ -0,0 +1,80 @@
+/// Statistics about packets transmitted on a connection
+#[derive(Default)]
+pub struct PacketTransmissionStats {
```

Comment:
I'd like to see all these structs marked `#[non_exhaustive]` to reflect the intention that more members will be added in the future.